dsalgo/doubly_linked_list.py

```diff
@@ -0,0 +1,54 @@
+class Node(object):
+    # Doubly linked node
+    def __init__(self, data=None, next=None, prev=None):
+        self.data = data
+        self.next = next
+        self.prev = prev
+
+class doubly_linked_list(object):
+    def __init__(self):
+        self.head = None
+        self.tail = None
+        self.count = 0
+
+    def append_item(self, data):
+        # Append an item 
+        new_item = Node(data, None, None)
+        if self.head is None:
+            self.head = new_item
+            self.tail = self.head
+        else:
+            new_item.prev = self.tail
+            self.tail.next = new_item
+            self.tail = new_item
+
+        self.count += 1
+
+    def insert_start(self, data):
+        #Insert item at the beginning
+        new_item = Node(data, None, None)
+        new_item.next = self.head
+        self.head = new_item
+        if self.tail is None:
+            self.tail = self.head
+        
+        self.count += 1
+
+
+    
+    def print_foward(self):
+        for node in self.iter():
+            print(node)
+            
+    def iter(self):
+        # Iterate the list
+        current = self.head
+        while current:
+            item_val = current.data
+            current = current.next
+            yield item_val
+
+# Use items = doubly_linked_list()
+# then items.append_item() append elements here any number of times
+# and use print_forward() to print the output as below :
+# items.print_foward()
```

Comment:
Add a blank line after each file

Reply:
Blank line, each file 🧐so I have to arrange comments with functions starting !?

Reply:
Can you check is it now correct @ https://github.com/codesankalp/dsalgo/pull/78/files#diff-c1dabacd7e9736a181ab83bf83dabbbb